registries/local/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Enable local registry
+echo "Installing a local registry"
+docker run -d -p 5000:5000 registry:2
+
+dev_ip=172.16.1.1
+sudo su -c "echo \"\" >> /etc/hosts"
+sudo su -c "echo \"$dev_ip       $IMAGE_REPOSITORY_HOST\" >> /etc/hosts"
+sudo ifconfig lo:0 $dev_ip
+kubectl create namespace riff
```

Comment:
Let's avoid calling things by the name of a different component in the system that it is not related to.